### PR TITLE
Pranita_Assignment_20_Done_15min+10min+30min+15min

### PR DESCRIPTION
**Pull request by PranitaPJun20**
armstrong
duplicate numb
missing number
reverse of num

### CODE REVIEW

File: src/pranitaPJun20/ArmstrongNumber.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+public class ArmstrongNumber {
+	//Method to find Armstrong Number
+	void findArmstrongNumber(int input) {
+		int numExpected= input;
+		int numActual=0;
+		while(input>0) {
+			int lastDigit  = input%10;
+			input=input/10;
+			numActual = numActual+(lastDigit*lastDigit*lastDigit);
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/ArmstrongNumber.java]: **Suggestion:**
You can take a look at power function - Math.pow(x,y)
which will help you achieve same result for multiplication of same number multiple times (to find square, cube and so on).
i.e. Math.pow((number % 10), 3))

File: src/pranitaPJun20/ArrayReverseNum.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+package pranitaPJun20;
+
+public class ArrayReverseNum {
+	// Method to find the reverse of the given number
+	void findReverseOfNumber(int number) {
+		int reverse = 0;
+		int input = number;
+		while (number > 0) {
+			int lastDigit = number % 10;
+			reverse = (reverse * 10) + lastDigit;
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/ArrayReverseNum.java:16]: No need to take extra variable. Can achieve same result by replacing equation of line 15 in line 16 for lastDigit.

File: src/pranitaPJun20/ArrayReverseNum.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ */
+package pranitaPJun20;
+
+public class ArrayReverseNum {
+	// Method to find the reverse of the given number
+	void findReverseOfNumber(int number) {
+		int reverse = 0;
+		int input = number;
+		while (number > 0) {
+			int lastDigit = number % 10;
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/ArrayReverseNum.java]: No need to take extra variable. Can achieve same result by replacing equation of line 15 in line 16 for lastDigit.

File: src/pranitaPJun20/ArrayDuplicateNum.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Program 2: Find duplicate number on Integer array in Java?
+Hint: There is exactly one number is repeated in the array. You need to write a program to
+find that duplicate number. For example, if an array with length 6 contains numbers {0, 3,
+1, 2, 3}, then duplicated number is 3.
+ */
+package pranitaPJun20;
+
+public class ArrayDuplicateNum {
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/ArrayDuplicateNum.java:9]: Perfect Logic.

File: src/pranitaPJun20/ArrayFindMissingNum20.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, 
+you need to write a Java program to find that missing number in an array.
+ */
+package pranitaPJun20;
+
+public class ArrayFindMissingNum20 {
+	int sumActual = 0;
```
Comment (AditiGJUN20) [inline src/pranitaPJun20/ArrayFindMissingNum20.java]: Variable needs to be declared as **local variable.**

**Review by AditiGJUN20 — CHANGES_REQUESTED** [review]
- Minor changes required.
- Avoid blank lines in between code.

File: src/pranitaPJun20/ArmstrongNumber.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/ArmstrongNumber.java]: Thank you Aditi.I came to know about new function.Changes Done.

File: src/pranitaPJun20/ArrayReverseNum.java (hunk 3 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/ArrayReverseNum.java]: Done.

File: src/pranitaPJun20/ArrayDuplicateNum.java (hunk 4 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/ArrayDuplicateNum.java:9]: Thank you :)

File: src/pranitaPJun20/ArrayFindMissingNum20.java (hunk 5 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/ArrayFindMissingNum20.java]: Yes correct.Made the change.

**Review by AditiGJUN20 — APPROVED** [review]
Changes are done as per comments. 
Code is approved.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Good job Pranita and Aditi.

**Pull request merged**